Sources/FingerprintJS/Harvesters/IdentifierHarvester/IdentifierHarvester.swift

```diff
@@ -0,0 +1,51 @@
+import UIKit
+
+protocol IdentifierHarvesting {
+    var vendorIdentifier: UUID? { get }
+}
+
+struct IdentifierHarvester {
+    private let vendorIdentifierLegacyKey = "vendorIdentifier"
+    private let vendorIdentifierKey = "vendorIdentifierBackground"
```

Comment:
The `Background` suffix in the key name might suggest that the value is only meant to be used by the tasks running in the background. How about using reverse-DNS notation and renaming the key to `id.vendor`?